client/config.go

```diff
@@ -58,6 +61,9 @@ type Config struct {
 	// See configureOAuth2 for more details.
 	OAuth2Config *OAuth2Config `yaml:"oauth2,omitempty"`
 
+	// IAPConfig is the Google Cloud Identity-Aware-Proxy configuration used for the client. (e.g. audience)
+	IAPConfig *IAPConfig `yaml:"iap,omitempty"`
```

Comment:
```suggestion
	IAPConfig *IAPConfig `yaml:"identity-aware-proxy,omitempty"`
```
IMO, identity-aware proxy is not sufficiently well-known for the `iap` acronym to be recognizable. 

Would it be possible to be a bit more explicit for the end users by using `identity-aware-proxy` in the YAML?

Reply:
Indeed, that's a good point. I've just changed the acronym in the readme, comments and yaml. I didn't change the variable names, as that would have been too verbose IMO.